src/test/functions/cache/hash.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from "chai";
+import { getLocalHash } from "../../../deploy/functions/cache/hash";
+import * as backend from "../../../deploy/functions/backend";
+import { resolve } from "path";
+
+const EMPTY_WANT_BACKEND = {
+  requiredAPIs: [],
+  environmentVariables: {},
+  endpoints: {},
+} as backend.Backend;
```

Comment:
nit: for variables you can just declare them to be the type you want instead of casting them.

Reply:
Good point! Updated (er, `empty()` makes the cast go away, but noted for everywhere else.)